app/views/support/_time_constraint.html.erb

```diff
@@ -22,6 +22,24 @@
       </span>
     </div>
 
+    <div class="form-group">
+      <span class="form-label">
+        <%= f.label :is_legal_deadline do %>
+          Is this a legal deadline?
+        <% end %>
+      </span>
+      <span class="form-wrapper">
+        <%= f.collection_radio_buttons :is_legal_deadline, f.object.is_legal_deadline_options, :last, :last, required: false do |builder| %>
```

Comment:
Is `required: false` necessary, or should this just be removed entirely? I imagine `false` is the default so doesn't need specifying?

EDIT: you remove it in `Validate times and radio button` 👍 So just needs removing from here too (the "Add radio button" commit)